lisa/operating_system.py

```diff
@@ -683,19 +683,20 @@ def get_repositories(self) -> List[RepositoryInfo]:
 
         return repositories
 
-    def add_repository(self, repo: str, key_location: str = "") -> None:
-        if key_location:
-            wget = self._node.tools[Wget]
-            key_file_path = wget.get(
-                url=key_location,
-                file_path=str(self._node.working_path),
-            )
-            self._node.execute(
-                cmd=f"apt-key add {key_file_path}",
-                sudo=True,
-                expected_exit_code=0,
-                expected_exit_code_failure_message="fail to add apt key",
-            )
+    def add_repository(self, repo: str, keys_location: List[str] = []) -> None:
```

Comment:
Make all method supports the same parameters, so it's easy to be data driven.

Reply:
They can have the same arguments, and not used. So the different os can use the same arg dict like we did for perf notifier.